test/test_ssltransport.py

```diff
@@ -27,7 +31,17 @@ def server_client_ssl_contexts():
     return server_context, client_context
 
 
-def sample_request(binary=True):
+@overload
+def sample_request(binary: "Literal[True]" = True) -> bytes:
```

Comment:
In overloads (and stub files) it is customary not to write `...` instead of the actual default values. This is because the default value itself is not meaningful at the type level, only that it exists, and this avoids duplication and is a bit shorter.